modules/ballerina-native/src/main/ballerina/ballerina/lang/jsons/natives.bal

```diff
@@ -68,3 +68,6 @@ native function toString (json j) (string);
 @doc:Param { value:"newKey: The new key value to use" }
 native function rename (json j, string jsonPath, string oldKey, string newKey);
 
+@doc:Description { value:"Parses and gets a JSON from a string."}
```

Comment:
You can reword this description as follows.

"Parses the string argument as a JSON value"